nl/tc_linux.go

```diff
@@ -1460,15 +1460,15 @@ func (p *TcPedit) SetIPv6Src(ip6 net.IP) {
 
 func (p *TcPedit) SetDstIP(ip net.IP) {
 	if ip.To4() != nil {
-		p.SetIPv4Dst(ip)
+		p.SetIPv4Dst(ip.To4())
```

Comment:
@Charlie17Li 
I think you need to bring back your original change along with the `To4()` modification.

`SetIPv4Dst` and `SetIPv4Src` are public methods. It is better the `To4()` change is done inside those two methods.

Reply:
ok